core/renderers/common/block_rendering.js

```diff
@@ -92,28 +92,28 @@ exports.unregister = unregister;
  * Turn on the blocks debugger.
  * @package
  * @alias Blockly.blockRendering.startDebugger
+ * @package
```

Comment:
Did you mean to create several duplicate `@package` annotations?  I am now getting what amount to lint errors from `npm run build:deps` as a result.